nixie/cli.py

```
@@ -200,28 +203,41 @@ async def render_block(
-        print(
-            "Error: '{0}' not found. Install Node.js with npx or Bun to use @mermaid-js/mermaid-cli.".format(
-                cli
-            ),
-            file=sys.stderr,
+        logging.getLogger(__name__).exception(
+            "Error: '%s' not found. Install Node.js with npx or Bun to use @mermaid-js/mermaid-cli.",
+            cli,
+        )
+    except RuntimeError:
+        logging.getLogger(__name__).exception(
+            "Runtime error while rendering diagram"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)